.github/workflows/ci.lint.yml

```diff
@@ -0,0 +1,41 @@
+name: Format & Lint
+
+on:
+  pull_request:
+  push:
+    branches:
+      - 'develop'
+
+jobs:
+  rust:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: "Setup Rust (+ Toolchain)"
+        run: |
+          curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs -y | sh
+          rustup update nightly
+          rustup component add rustfmt --toolchain nightly-x86_64-unknown-linux-gnu
+
+      - name: "Format"
+        run: yarn lint:rust
```

Comment:
```suggestion
      - name: Format
         run: cargo +nightly fmt -- --check
         working-directory: packages/backend
```
`rustfmt` is a bit odd and works the opposite way of `eslint` and `prettier`. `eslint` and `prettier` check when called without arguments, and require `--fix` or `-w` (respectively) to automatically fix files, but `rustfmt` actually fixes the files when called without arguments (and exits with an exit code of 0 even if it had to fix something). `-- --check` needs to be added if we only want to check the formatting (and exit with a nonzero code if changes need to be made), which is best for CI:

```
➜  backend git:(develop) cargo +nightly fmt -- --check
Diff in /Users/rajiv/firefly/packages/backend/src/actors.rs at line 9:
 use riker::actors::*;
 
 use super::RUNTIME;
-use std::{sync::Arc};
+use std::sync::Arc;
 #[derive(Clone, Debug)]
 pub struct KillMessage;
 
Diff in /Users/rajiv/firefly/packages/backend/src/lib.rs at line 9:
     client::drop_all as drop_clients,
     event::{
         on_balance_change, on_broadcast, on_confirmation_state_change, on_error,
-        on_migration_progress, on_new_transaction, on_reattachment, on_stronghold_status_change,
-        on_transfer_progress, on_ledger_address_generation, remove_balance_change_listener, remove_broadcast_listener,
-        remove_confirmation_state_change_listener, remove_error_listener,
+        on_ledger_address_generation, on_migration_progress, on_new_transaction, on_reattachment,
+        on_stronghold_status_change, on_transfer_progress, remove_balance_change_listener,
+        remove_broadcast_listener, remove_confirmation_state_change_listener,
+        remove_error_listener, remove_ledger_address_generation_listener,
         remove_migration_progress_listener, remove_new_transaction_listener,
         remove_reattachment_listener, remove_stronghold_status_change_listener,
-        remove_transfer_progress_listener, remove_ledger_address_generation_listener, EventId,
+        remove_transfer_progress_listener, EventId,
     },
 };
 use once_cell::sync::Lazy;
Diff in /Users/rajiv/firefly/packages/backend/src/lib.rs at line 167:
                 remove_stronghold_status_change_listener(event_id).await
             }
             EventType::TransferProgress => remove_transfer_progress_listener(event_id).await,
-            EventType::LedgerAddressGeneration => remove_ledger_address_generation_listener(event_id).await,
+            EventType::LedgerAddressGeneration => {
+                remove_ledger_address_generation_listener(event_id).await
+            }
             EventType::MigrationProgress => remove_migration_progress_listener(event_id).await,
         };
     }
```